core/src/test/java/io/github/tcdl/ChannelManagerTest.java

```diff
@@ -106,7 +106,7 @@ public void testReceiveMessageInvokesAgentAndEmitsEvent() throws InterruptedExce
             awaitReceiveEvents.countDown();
         });
 
-        assertTrue(awaitReceiveEvents.await(3000, TimeUnit.MILLISECONDS));
+        assertTrue(awaitReceiveEvents.await(4000, TimeUnit.MILLISECONDS));
```

Comment:
I assume these changes are not related to Jacoco plugin. Please commit them separately